pkg/cmd/cli/plugin/add.go

```diff
@@ -131,7 +131,7 @@ func NewAddCommand(f client.Factory) *cobra.Command {
 			patchBytes, err := strategicpatch.CreateTwoWayMergePatch(original, updated, v1beta1.Deployment{})
 			cmd.CheckError(err)
 
-			_, err = kubeClient.AppsV1beta1().Deployments(ark.DefaultNamespace).Patch(arkDeploy.Name, types.StrategicMergePatchType, patchBytes)
+			_, err = kubeClient.AppsV1beta1().Deployments(ark.DefaultNamespace).Patch(arkDeploy.Name, types.MergePatchType, patchBytes)
```

Comment:
Why the change?

Reply:
I guess it's not actually necessary; saw that strategic merge patch doesn't work OOTB for CRD's so switched those to `MergePatchType` and changed this one to be consistent. It works either way given what we're doing, but I can leave it as `StrategicMergePatchType`.